include/deal.II/grid/tria_accessor.h

```diff
@@ -2766,8 +2835,7 @@ class CellAccessor : public TriaAccessor<dim, dim, spacedim>
   /**
    * Return an array of iterators to all faces of this cell.
    */
-  std::array<TriaIterator<TriaAccessor<dim - 1, dim, spacedim>>,
-             GeometryInfo<dim>::faces_per_cell>
+  std::vector<TriaIterator<TriaAccessor<dim - 1, dim, spacedim>>>
   face_iterators() const;
```

Comment:
That's unfortunate since it requires dynamic allocation. What if you used `boost::container::small_vector` instead with a fixed size of `2^dim`?

Reply:
I like this solution! Thanks!